views/js/editProfile.js

```diff
@@ -1,14 +1,14 @@
 function init() {
     var currUser;
     $.ajax({
-        url: 'http://localhost:8080/',
+        url: 'https://localhost:8080/getCurrUser',
         type: 'GET',
-        dataType: 'application/json',
+        dataType: 'json',
         complete: function (data) {
             alert(data)
         },
         success: function (data) {
-            currUser = JSON.parse(data);
+            currUser = data;
```

Comment:
Verification of data is also to be done here whether the data you are geting is correct or not.

Reply:
Verification of data and fields in it are pending.